docs/Using Fleet/Fleet-UI.md

```diff
@@ -46,41 +46,21 @@ The query may take several seconds to complete because Fleet has to wait for the
 
 *In Fleet 4.35.0, the "Schedule" page was removed, and query automations are now configured on the "Queries" page. Instructions for scheduling queries in earlier versions of Fleet can be found [here](https://github.com/fleetdm/fleet/blob/ac797c8f81ede770853c25fd04102da9f5e109bf/docs/Using-Fleet/Fleet-UI.md#schedule-a-query).*
 
->Only users with the [admin role](https://fleetdm.com/docs/using-fleet/manage-access#admin) can manage query automations.
+Fleet allows you to schedule queries to run at a set frequency. Scheduled queries will send data to Fleet and/or your [log destination](https://fleetdm.com/docs/using-fleet/log-destinations) automatically. 
 
-Fleet allows you to schedule queries to run at a set frequency. Scheduled queries will send data to your log destination automatically. 
+By default, queries that run on a schedule will only target platforms compatible with that query. This behavior can be overridden by setting the platforms in **Advanced options** when saving a query.
```

Comment:
```suggestion
By default, queries that run on a schedule will only target platforms compatible with that query. This behavior can be overridden by setting the platforms in "**Advanced options**" when saving a query.
```
Does this need quotes?

Reply:
@Sampfluger88 it's a little all-over-the-place still, but: awhile back Noah & Marko & I landed on the convention of bolding UI references in the docs or in the product. Example:
<img width="535" alt="Screenshot 2023-11-09 at 1 50 59 PM" src="https://github.com/fleetdm/fleet/assets/3065949/af8fa829-3c7d-4736-af50-577dc969c61e">